docs/source/development-guide/style-guide/git-and-github-workflow.rst

```diff
@@ -59,26 +64,33 @@ well.*
 
    .. code-block:: bash
 
-       pip install .[dev]
-       pre-commit install
+       pip3 install pre-commit
+
+   If this results in errors, check the correct form of pip is being employed using ``which pip``.
 
 #. Retrieve the ``upstream`` ``dev`` branch:
 
    .. code-block:: bash
 
        git fetch upstream
-       git checkout -b dev upstream/dev
+       git checkout -b dev upstream-dev
+
+   To check this step, run ``git status``. The branch name should be ``upstream-dev``.
 
 #. Create a feature branch off of the ``dev`` branch to develop changes on. Branch names should be short but
    descriptive (e.g. ``update-codice-unit-tests``) and not too generic (e.g. ``bug-fix``, ``updates``). Consistent use
    of hyphens is encouraged.
 
+   Place the terminal in the file path of the fork you created in step one. Then, run the following command:
+
    .. code-block:: bash
 
         git checkout -b <branchname>
 
 #. Make changes to the branch using the nominal ``git add``/``git commit`` cycle.
 
+   *Note: Open the forked repository in a chosen IDE to make alterations.*
```

Comment:
Slight change to the wording here to help make it clear that its not necessarily _required_ to use an IDE

```suggestion
   *Note: You can open the forked repository in a chosen IDE to make changes.*
```